draft-packages/zen-navigation-bar/KaizenDraft/ZenNavigationBar/components/Badge.module.scss

```diff
@@ -1,7 +1,7 @@
 @import "~@kaizen/design-tokens/sass/color";
 @import "~@kaizen/component-library/styles/color";
-@import "~@kaizen/component-library/styles/type";
 @import "~@kaizen/component-library/styles/layout";
+@import "~@kaizen/component-library/styles/type";
```

Comment:
While we're here, this `type.scss` is actually deprecated.

For new typography stuff, there's `~@kaizen/design-tokens/sass/typography`.

For `$ca-grid`, there's `~@kaizen/component-library/styles/grid`.

I see there are things like `$ca-weight-medium` in here that still depend on `type` so no need to get rid of them now, but just an FYI to spread awareness 👍